docs/workshop/sql_query_engine.md

```diff
@@ -1,3 +1,38 @@
-# SQL Query Engine - Trino
+# Create SQL tables from pull requests dataset
 
-This section will include instructions for sending the data via Trino notebook, accessing and interacting with the dataset through Cloudbeaver UI.
+We can also create interactive visualizations and charts on [Apache Superset](https://superset.apache.org/) using the data that we collect from Github Repositories.
```

Comment:
Should we add the opf workshop url for Superset here to avoid confusion for folks?